frontend/src/App.js

```diff
@@ -47,28 +50,27 @@ import User from './components/AdminControls/User.js'
 //     text: "I am the Cat Religion's text!",
 //   }
 // ]
-
-class App extends Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      message: 'Loading...'
-    }
-  }
-  componentDidMount() {
-    axios.get('/api/message.json')
-    .then(response => {
-      this.setState({ message: response.data.message })
-    })
-    .catch(console.error)
+ 
+ class App extends Component {
+   constructor(props) {
+     super(props)
+     this.state = {
+       message: 'Loading...'
+     }
+   }
+   componentDidMount() {
+     axios.get('/api/message.json')
+     .then(response => {
+       this.setState({ message: response.data.message })
+     })
+     .catch(console.error)
```

Comment:
I think you can remove the extra space in front here. The `class` definition should be on the left margin, and two-space-wide tab.